src/datreant/core/util.py

```diff
@@ -17,3 +20,35 @@ def makedirs(path):
             pass
         else:
             raise
+
+
+def touch_me(path):
```

Comment:
I was more asking about what `touch_me` does. It looks to me like it does the same thing as the code it is intended to replace. What's the point?